frontend/src/Editor/Inspector/Components/Table.jsx

```diff
@@ -783,6 +783,7 @@ class TableComponent extends React.Component {
     if (!component.component.definition.properties.displaySearchBox)
       paramUpdated({ name: 'displaySearchBox' }, 'value', true, 'properties');
     const displaySearchBox = component.component.definition.properties.displaySearchBox.value;
+    const displayServerSideSearch = displaySearchBox === "{{false}}" ? false : true;
```

Comment:
Hey @tanishk-23 , Please resolve the value of `displaySearchBox` like this: `resolveReferences(component.component.definition.properties.displaySearchBox?.value, currentState)`

to get `displayServerSideSearch` instead of comparing it against `{{false}}`. (Because this value could easily be something like `{{components.toggle1.value}}`